### ISSUE TITLE
Parsing error with "read, write"

### ISSUE THREAD

**Issue by SamyPesse** [brianneisler/firemin#36]
First of all thank you for building this tool 🙌  We are seriously impacted by the 400 error during Firebase deployment and minifying our rules is our only viable option.

We've noticed an issue with case like: `allow read, write: if request.auth != null;` (multiple operations separated by commas).

You can reproduce it with:

```
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if request.auth != null;
    }
  }
}
```

```
[1620131069589] WARN (22275 on MacBook-Pro-de-Samy.local): An error occured while trying to minimize the rules file
[1620131069591] WARN (22275 on MacBook-Pro-de-Samy.local): pino.final with prettyPrint does not support flushing
Error: Expected operator ':'. Instead was given ',' at 4:16
    at Object.parse (/Users/samypesse/Projects/squad/node_modules/firetree/src/parser/nodes/ColonOperator.js:20:13)
    at parseColonOperator (/Users/samypesse/Projects/squad/node_modules/firetree/src/parser/pipes/parseColonOperator.js:7:34)
    at /Users/samypesse/Projects/squad/node_modules/ramda/src/internal/_pipe.js:3:14
    at /Users/samypesse/Projects/squad/node_modules/ramda/src/internal/_pipe.js:3:27
    at /Users/samypesse/Projects/squad/node_modules/ramda/src/internal/_pipe.js:3:27
    at /Users/samypesse/Projects/squad/node_modules/ramda/src/internal/_pipe.js:3:27
    at parseAllowStatementTokens (/Users/samypesse/Projects/squad/node_modules/ramda/src/internal/_arity.js:11:19)
    at Object.parse (/Users/samypesse/Projects/squad/node_modules/firetree/src/parser/nodes/AllowStatement.js:44:34)
    at /Users/samypesse/Projects/squad/node_modules/firetree/src/parser/util/parseNextNode.js:19:21
    at /Users/samypesse/Projects/squad/node_modules/ramda/src/internal/_curryN.js:46:27
```

**Comment by brianneisler** [comment]
@SamyPesse, thanks for reporting! Should be an easy fix. Will look at patching this up 👍

**Comment by brianneisler** [comment]
@SamyPesse, this has been fixed in release v0.3.1 https://github.com/brianneisler/firemin/pull/38

Thanks for reporting! please let me know if you run into any other issues with the minifier or have any feature requests.
